### DISCUSSION TITLE
LabA-E spec詢問

### DISCUSSION THREAD

**Discussion by Lin-Xiang88** [bol-edu/HLS-SOC-Discussions#149] [Q&A]
想詢問 是否有Lab A-E的詳細資訊，方便學生實作
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/43726100/72c96586-8489-4949-ae01-42e1c3d27444)

**Comment by ZheChen-Bill** [comment]
因為LAB#6有很大部分的內容與LAB#A~E重複，因此有更改LAB的時程與方式，同學應該可以在最新的行事曆看到更新。
